volume/volume.go

```diff
@@ -156,6 +156,10 @@ func (m *MountPoint) Setup(mountLabel string, rootIDs idtools.IDPair, checkFun f
 		if err == nil {
 			if label.RelabelNeeded(m.Mode) {
 				if err = label.Relabel(m.Source, mountLabel, label.IsShared(m.Mode)); err != nil {
+					if err == syscall.ENOTSUP {
```

Comment:
Not fond on the `switch`, but nesting can be improved; perhaps something like this?

```go
defer func() {
	if err != nil {
		return
	}
	if !label.RelabelNeeded(m.Mode) {
		return
	}
	if err = label.Relabel(m.Source, mountLabel, label.IsShared(m.Mode)); err != nil {
		if err == syscall.ENOTSUP {
			err = nil
			return
		}
		path = ""
		err = errors.Wrapf(err, "error setting label on mount source '%s'", m.Source)
	}
}()
```